### DISCUSSION TITLE
Disable Keybind

### DISCUSSION THREAD

**Discussion by ZarosRiel** [T-Troll/alienfx-tools#233] [Q&A]
Hello,
I'm using fan control most of the time and i can't seem to figure out how to disable the keybinds because when the application is running i cannot use my CTRL+ALT+"....".
Thank you.

**Comment by T-Troll** [comment]
It's not possible to do it right now.
I'll add an option to disable it into the next release.

**Reply by ZarosRiel** [reply]
Ah alright. Thank you.

**Reply by T-Troll** [reply]
Check [7.4.5](https://github.com/T-Troll/alienfx-tools/releases/tag/7.4.5)
